src/main/java/org/threeten/extra/scale/UtcRules.java

```diff
@@ -226,6 +226,70 @@ public TaiInstant convertToTai(UtcInstant utcInstant) {
      */
     public abstract UtcInstant convertToUtc(TaiInstant taiInstant);
 
+    /**
```

Comment:
Having thought a bit, my view is that we should make UTC and TAI the primary time scales, and others like MISP secondary. This avoids the risk that `UtcRules` ends up with conversion methods for all possible time scales (n x n combination problem).

As such, please move these methods to `MispInstant` (as necessary). Conversion from MISP to UTC will need to go via `TaiInstant`

Reply:
Still to be addressed, but I think I understand what is needed.